src/pages/index.js

```diff
@@ -170,14 +293,27 @@ previewModalExitButton.addEventListener("click", () => {
   closeModal(previewModal);
 });
 
-editFormElement.addEventListener("submit", handleEditFormSubmit);
-cardFormElement.addEventListener("submit", handleAddCardSubmit);
+avatarEditButton.addEventListener("click", () => {
+  openModal(avatarModal);
+});
 
-initialCards.forEach((item) => {
-  const cardElement = getCardElement(item);
-  cardsList.prepend(cardElement);
+avatarExitButton.addEventListener("click", () => {
+  closeModal(avatarModal);
 });
 
+deleteCardExitButton.addEventListener("click", () => {
+  closeModal(deteleCardModal);
+});
+
+deleteCardCancelButton.addEventListener("click", () => {
+  closeModal(deteleCardModal);
+});
+
+editFormElement.addEventListener("submit", handleEditFormSubmit);
+cardFormElement.addEventListener("submit", handleAddCardSubmit);
+avatarFormElement.addEventListener("submit", handleAvatarFormSubmit);
+deleteCardFormElement.addEventListener("submit", handleDeleteCardSubmit);
+
 [editProfileModal, editCardModal, previewModal].forEach((modal) => {
```

Comment:
* The `confirmation` modal should close on `overlay` click as well
* The `avatar` modal should close on `overlay` click as well

Please, add them both into the array of modals inl ine 317


```js

[editProfileModal, editCardModal, previewModal, deteleCardModal, avatarModal].forEach((modal) => {

```